content/05.main-text.md

```diff
@@ -1,15 +1,53 @@
 ## Introduction {.page_break_before}
 
+### Hetionet
```

Comment:
I'm guessing that you are using this a short-hand for a broader discussion point about network approaches and heterogeneous networks, but if not you probably want to start a bit broader :)

Reply:
The intro headings are incomplete, and we may decide not to have headings here at all. But this is a placeholder to make sure we introduce hetionet as that's the hetnet we apply the approach to. Before that we should introduce hetnets generally.